src/org/zaproxy/zap/extension/exportreport/FileChooser/ReportFileView.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ * This file is based on the Paros code file ReportLastScan.java
+ */
+package org.zaproxy.zap.extension.exportreport.FileChooser;
+
+import java.io.File;
+import javax.swing.Icon;
+import javax.swing.filechooser.FileView;
+
+/*
+ * AUTHOR		:	GORAN SARENKAPA - JordanGS
+ * SPONSOR		:	RYERSON UNIVERSITY
+ */
+
+public class ReportFileView extends FileView
+{
+	FileList	list;
+
+	public ReportFileView(FileList list)
+	{
+		this.list = list;
+	}
+
+	public String getName(File f)
+	{
+		return null; // let the L&F FileView figure this out
+	}
+
+	public String getDescription(File f)
+	{
+		return null; // let the L&F FileView figure this out
+	}
+
+	public Boolean isTraversable(File f)
+	{
+		return null; // let the L&F FileView figure this out
+	}
+
+	// Not currently used for anything but can be in future.
```

Comment:
Was referring specifically to the comment, it's the same behaviour as other methods (which do not have a comment).